tests/test_home_page.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from utilities.config_reader import ReadConfig
+from page_objects.login_page import LoginPage
+from page_objects.home_page import HomePage
+
+
+@pytest.mark.smoke
+def test_suggested_pages_count(initialize_driver):
+    email = ReadConfig.get_user_email()
+    password = ReadConfig.get_user_password()
+    driver = initialize_driver
+    suggested_pages_url = ReadConfig.get_suggested_pages_url()
+    (LoginPage(driver).set_email(email).set_password(password).
+     click_login_button().click_save_info_not_now_button().
+     click_notification_not_now_button().click_see_all_button().
+     wait_url_to_be(suggested_pages_url))
+    last_suggested_page = HomePage(driver).last_suggested_page_is_displayed()
+    assert last_suggested_page
```

Comment:
???
Where is assertion message? add it all around project. 
Also, your test should verify several attributes. What if element is displayed but with the text "FUCK OUR CUSTOMERS". Your test will pass and will not find a bug. Rethink all tests around project

Reply:
@bohdandovbsyh this case checks the amount of RECOMMENDED pages displayed on Instagram.
I don't care what Instagram suggests or how the pages are named.